src/components/Landing/BurnerNotConnected.tsx

```diff
@@ -0,0 +1,37 @@
+import { HeaderText, SubHeaderText } from 'components/ui/Text'
+import SealGrid from 'icons/SealGrid'
+import classnames, {
+  display,
+  justifyContent,
+  space,
+  textAlign,
+  width,
+} from 'classnames/tailwind'
+
+const sealGridWrapper = classnames(
+  display('flex'),
+  justifyContent('justify-center'),
+  width('w-60', 'sm:w-72', 'md:w-seal-grid')
+)
+const headerTextWrapper = classnames(
+  space('space-y-2'),
+  textAlign('text-center')
+)
+
+export default function () {
```

Comment:
This component looks pretty similiar to `BurnerNotCorrect`, maybe we can have only one component with 2 props: `headerText, subheaderText`?

Reply:
Mixed them